balboa-agent/src/main/scala/com/socrata/balboa/agent/Config.scala

```diff
@@ -29,6 +36,18 @@ trait Config extends JMSClientConfig {
 
   def initialDelay(defaultDelay: Long = 0): Long = Configuration.get().getLong(Keys.InitialDelay, defaultDelay)
 
+  def httpOrMQ(defaultMethod: HttpOrMq = HttpOrMq.HTTP): HttpOrMq =
+    Configuration.get().getString(Keys.HttpOrMq, defaultMethod.toString) match {
+      case "HTTP" => HttpOrMq.HTTP
+      case "MQ" => HttpOrMq.MQ
+      case _ => defaultMethod
+    }
+
+  def balboaHttpUrl(defaultUrl: String): String =
```

Comment:
It looks like you added this method, but the only call site I can find has an empty string as the default value. That pretty much means there doesn't need to be a default value parameter.

What about returning an Option, and if None or empty, log an error and exit. I can't think of a way to make it a recoverable scenario if there is no balboa-http endpoint specified.